ee/secureenclaverunner/secureenclaverunner.go

```diff
@@ -231,8 +287,8 @@ func (ser *secureEnclaveRunner) currentConsoleUserKey(ctx context.Context) (*ecd
 func (ser *secureEnclaveRunner) MarshalJSON() ([]byte, error) {
 	keyMap := make(map[string]string)
 
-	for uid, pubKey := range ser.uidPubKeyMap {
-		pubKeyBytes, err := x509.MarshalPKIXPublicKey(pubKey)
+	for uid, entry := range ser.uidPubKeyMap {
+		pubKeyBytes, err := x509.MarshalPKIXPublicKey(entry.pubKey)
```

Comment:
Maybe worth documenting here also (like you do in the unmarshal function) that we don't want to include `entry.verifiedInSecureEnclave` when marshalling because we don't want to save that info in the store, so that verified state won't get copied if the launcher db gets copied to a new machine?

Reply:
👍  added a comment